client/src/index.html

```diff
@@ -4,14 +4,24 @@
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="initial-scale=1, width=device-width" />
 		<title>Starter Kit</title>
+		<title>Module Deadline Tracker</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
 	</head>
 	<body>
 		<noscript>You need to enable JavaScript to run this app.</noscript>
 		<div id="root"></div>
-		<script
+		<h1>Module Deadline Tracker</h1>
+  <div id="module-info">
+    <h2 id="module-name">Module Name: </h2>
+    <h2 id="deadline">Deadline: </h2>
+    <h2 id="remaining-days">Remaining Days: </h2>
+  </div>
```

Comment:
This should all be within the component. You probably wont have a need to edit this file for this project. Anything you do add here will be on every page and outside React